lib/v2/datasource/remote/api/guardians_repository.dart

```diff
@@ -169,6 +170,30 @@ class GuardiansRepository extends EosRepository with NetworkRepository {
             }))
         .catchError((error) => mapEosError(error));
   }
+
+  Future<Result<dynamic>> getAccountRecovery(String accountName) async {
+    print('[http] get account guardians');
+    var code = 'guard.seeds';
+    var scope = 'guard.seeds';
+    var table = 'guards';
```

Comment:
These strings should be in the app_constans?

Reply:
https://github.com/JoinSEEDS/seeds_light_wallet/pull/787

Once this is merged, 
ill fix these constants